generators/getMetric.js

```diff
@@ -0,0 +1,86 @@
+const path = require('path')
+const fs = require('fs')
+const { metricsList } = require('./getMetricFunctions')
+const fsExtra = require('fs-extra')
+
+function generateGetMetric () {
+  let generatedDoc = ''
+  for (const metric of metricsList) {
+    generatedDoc += allTimeBoundTemplates(metric)
+  }
+  let file = path.join(`${__dirname}`, `../src/getMetric.js`)
+  if (fs.existsSync(file)) {
+    fsExtra.removeSync(file)
+  }
+  return fs.writeFileSync(file, generatedDoc)
+}
+
+function allTimeBoundTemplates (metric) {
+  if (!('currency' in metric)) {
+    metric.currency = ''
+  }
+  if (!('sheetMetricName' in metric)) {
+    metric.sheetMetricName = metric.metric
+  }
+  if (!('timeBound' in metric)) {
+    metric.timeBound = ''
+  }
+  let generatedString =
+    fillTemplate(metric.metric, metric.sheetMetricName, metric.description, metric.currency, metric.timeBound)
+
+  return generatedString
+}
+
+function fillTemplate (metric, sheetMetricName, description, currency, timeBound) {
+  let [options, bonusDescription, functionArguments] =
+    optionsGenerator(currency, timeBound)
+  return `/**
+* Gets ${description}
+* @param {string} projectSlug Name of the asset at sanbase,
+* which can be found at the end of the URL (eg. the Santiment URL is
+* https://app.santiment.net/projects/santiment, so the projectSlug would be santiment).
+* @param {date} from The starting date to fetch the data. Example: DATE(2018, 9, 20)
```

Comment:
Let's remove `Example` from the doc here

Reply:
Woulnd't it be better if we were consistent? In the file sheetMetrics.js there are Examples, both for from and to, just like in here.

Reply:
Yep, let's keep the example, we have it for every function.